### PR TITLE
Update to EISOP 3.41-eisop1 from 3.40-eisop2 

### PR DESCRIPTION
**Pull request by Ao-senXiong**
Merge with:
https://github.com/opprop/jdk/pull/8
https://github.com/opprop/annotation-tools/pull/134
https://github.com/opprop/checker-framework-inference/pull/447

### CODE REVIEW

**Comment by wmdietl** [comment]
`junit_jdk21` is consistently failing with:

````
org.checkerframework.checker.test.junit.NullnessTest > run[/__w/1/s/checker/tests/all-systems/java21] FAILED
    java.lang.AssertionError: 0 out of 0 expected diagnostics were found.
    6 unexpected diagnostics were found:
      Issue6173.java:16: warning: (nulltest.redundant)
      JEP441.java:58: warning: (nulltest.redundant)
      JEP441.java:66: warning: (nulltest.redundant)
      JEP441.java:80: warning: (nulltest.redundant)
      JEP441.java:97: warning: (nulltest.redundant)
      JEP441.java:191: warning: (nulltest.redundant)
````

Is that not happening for you locally? Is some branch misnamed?

**Comment by Ao-senXiong** [comment]
> `junit_jdk21` is consistently failing with:
> 
> ```
> org.checkerframework.checker.test.junit.NullnessTest > run[/__w/1/s/checker/tests/all-systems/java21] FAILED
>     java.lang.AssertionError: 0 out of 0 expected diagnostics were found.
>     6 unexpected diagnostics were found:
>       Issue6173.java:16: warning: (nulltest.redundant)
>       JEP441.java:58: warning: (nulltest.redundant)
>       JEP441.java:66: warning: (nulltest.redundant)
>       JEP441.java:80: warning: (nulltest.redundant)
>       JEP441.java:97: warning: (nulltest.redundant)
>       JEP441.java:191: warning: (nulltest.redundant)
> ```
> 
> Is that not happening for you locally? Is some branch misnamed?

@wmdietl Looks the symbolic link is not created in opprop and all-systems tests case did not update the change. I just created it.

File: framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java [hunk 1]
```diff
@@ -446,7 +446,7 @@ private final AnnotationMirror lowestQualifier(
      * instead of this method.
      */
     @Override
-    public final V leastUpperBound(@Nullable V other) {
+    public V leastUpperBound(@Nullable V other) {
```
Comment (wmdietl) [inline framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java:449]: Can't CFI override the method mentioned in the javadoc instead?

Comment (Ao-senXiong) [reply framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java]: Yes, I think it can. I just override the commits.

**Pull request merged**